tetris.py

```diff
@@ -59,82 +84,98 @@ def __init__(self, speed, canvas):
         self.canvas = canvas
         self.rect_size = 25
         self.tetris_game = TetrisGame()
```

Comment:
To do 

```
self.main_board.draw_block(self.tetris_game.whatever)
```

like we discussed, you need to change TetrisGui's `__init__` like this (can't do a suggestion for whatever reason):

```
def __init__(self, speed, main_board):
    self.speed = speed
    self.main_board = main_board
    ...
```

Reply:
Done!